model/src/pyrenew/convolve.py

```diff
@@ -17,62 +17,110 @@
 
 import jax.numpy as jnp
 from jax.typing import ArrayLike
+from pyrenew.transformation import IdentityTransform
 
 
-def new_convolve_scanner(discrete_dist_flipped: ArrayLike) -> Callable:
-    """
-    Factory function to create a scanner function for convolving a discrete distribution
-    over a time series data subset.
+def new_convolve_scanner(
+    discrete_dist_flipped: ArrayLike, transform: Callable = None
+) -> Callable:
+    r"""
+    Factory function to create a "scanner" function
+    that can be used with :py:func:`jax.lax.scan` to
+    construct an array via backward-looking iterative
+    convolution.
 
     Parameters
     ----------
     discrete_dist_flipped : ArrayLike
-        A 1D jax array representing the discrete distribution flipped for convolution.
+        A 1D jax array representing the discrete
+        distribution flipped for convolution.
+
+    transform : Callable
+        A transformation to apply to the result
+        of the dot product and multiplication.
+        If None, use the identity transformation.
+        Default None.
 
     Returns
     -------
     Callable
-        A scanner function that can be used with jax.lax.scan for convolution.
-        This function takes a history subset and a multiplier, computes the dot product,
-        then updates and returns the new history subset and the convolution result.
+        A scanner function that can be used with
+        jax.lax.scan for convolution.
+        This function takes a history subset and
+        a multiplier, computes the dot product,
+        then updates and returns the new history
+        subset and the convolution result.
+
+    Notes
+    -----
+    The following iterative operation is found often
+    in renewal processes:
+
+    .. math::
+        X(t) = f\left(m(t) * \left[X(t - n),
+        X(t - n + 1), ... X(t - 1)\right] \dot \vec{d} \right)
+
+    Where `math`:\vec{d} is a vector of length `math`:n,
+    `math`:m(t) is a scalar for each value of time `math`:t,
+    and `math`:f is a scalar-valued function.
+
+    Given `math`:\vec{d}, and optionally `math`:f,
+    this factory function returns a new function that
+    peforms one step of this process while scanning along
+    an array of  multipliers (i.e. an array
+    giving the values of `math`:m(t)) using :py:func:jax.lax.scan.
```

Comment:
This section is not rendering properly on the website:

![image](https://github.com/CDCgov/multisignal-epi-inference/assets/893619/5f3c2ea8-f351-4cd9-9503-091854855d3b)

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/161/commits/b3e8d98f127323de17b3dfef089429f814dfb54f

Reply:
Thanks, apologies for the bad sphinx